flow/connectors/mysql/cdc.go

```diff
@@ -298,6 +303,12 @@ func (c *MySqlConnector) PullRecords(
 		}
 
 		switch ev := event.Event.(type) {
+		case *replication.QueryEvent:
+			if string(ev.Query) == "COMMIT" {
```

Comment:
what about `ROLLBACK`?
also `COMMIT WORK` and other MySQLisms: https://mariadb.com/kb/en/start-transaction/

Reply:
rolled back transactions should not appear in binlog (otherwise we'd need to support rollback like in pg v2)

Reply:
not even going to punt on this for now, removed